CHANGELOG.md

```diff
@@ -33,12 +32,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - **BREAKING**: `PdfLoader` no longer accepts `str` file content, `Path` file paths or `IO` objects as sources. Instead, it will only accept the content of the PDF file as a `bytes` object.
 - **BREAKING**: `TextLoader` no longer accepts `Path` file paths as a source. It will now accept the content of the text file as a `str` or `bytes` object.
 - **BREAKING**: `FileManager.default_loader` is now `None` by default.
-- **BREAKING**: Replaced `EventListener.handler` with `EventListener.driver` and `LocalEventListenerDriver`.
 - **BREAKING** Bumped `pinecone` from `^2` to `^3`.
 - **BREAKING**: Removed `workdir`, `loaders`, `default_loader`, and `save_file_encoding` fields from `FileManager` and added `file_manager_driver`.
 - **BREADKING**: Removed `mime_type` field from `ImageArtifact`. `mime_type` is now a property constructed using the Artifact type and `format` field.
 - Improved RAG performance in `VectorQueryEngine`.
 - Moved [Griptape Docs](https://github.com/griptape-ai/griptape-docs) to this repository.
+- The return value of `EventListener.handler` will be passed to the `EventListenerDriver.try_publish_event_payload`'s' `event_payload` parameter.
 
 ### Fixed
 - Type hint for parameter `azure_ad_token_provider` on Azure OpenAI drivers to `Optional[Callable[[], str]]`.
```

Comment:
Maybe "CHANGED" type hint? Can we clarify what this fixed?
Ditto for line 45 below. Did we SUPPLY the missing params?

Reply:
This didn't fix anything, it updated the behavior of the `handler` function to use the user's return value in the Driver. Previously this return value was thrown away.

Reply:
> Ditto for line 45 below.

Assuming you're talking about `Missing parameters azure_ad_token and azure_ad_token_provider`, this _was_ a fix because the type hints were incorrect.

Reply:
Yes, it's the missing parameters here, I was advocating for including a verb in the sentence, or at least clarifying what was wrong and what was done to fix it.